### DISCUSSION TITLE
Missing Pin Information

### DISCUSSION THREAD

**Discussion by hdpreece** [judge2005/BambuLights#3] [Q&A] [answered]
I struggled to find what pin the LED data pin should connect to on the microcontroller. Any ideas? It would be really cool to set that  through the web interface.

**Comment by judge2005** [comment, marked as answer]
It’s connected to IO27, but I’m not sure I’m answering your question here.

I’ll think about being able to set it via web interface.

**Reply by GerNavBet** [reply]
Thanks for such project. Is it possible to modify the Signal Pin #? Thanks.

**Reply by GerNavBet** [reply]
Installed it on a ESP32S but can manage to get the correct Pin for data. Would you be able to guide me though this? Thanks.

**Reply by judge2005** [reply]
according to this IO27 is available on pin 11: https://forum.fritzing.org/t/esp32s-hiletgo-dev-boad-with-pinout-template/5357. This may not be the module you are using, but you should be able to find the pin outs for your module somewhere.
